stats/user_logon_stats.php

```diff
@@ -2,21 +2,26 @@
 $relPath = './../pinc/';
 include_once($relPath.'base.inc');
 include_once($relPath.'theme.inc');
+include_once($relPath.'graph_data.inc');
 
 require_login();
 
 $title = _("User Logon Statistics");
-output_header($title);
+output_header($title, SHOW_STATSBAR, [
+    "js_files" => get_graph_js_files(),
+    "js_data" => '$(function(){
+  stackedAreaChart("past_year_preceding_hour", ' . json_encode(user_logging_on("year", "hour")) . ');
+  stackedAreaChart("past_year_preceding_day", ' . json_encode(user_logging_on("year", "day")) . ');
+  stackedAreaChart("past_year_preceding_week", ' . json_encode(user_logging_on("year", "week")) . ');
+  stackedAreaChart("past_year_preceding_fourweek", ' . json_encode(user_logging_on("year", "fourweek")) . ');
+});',
+]);
 echo "<h1>$title</h1>";
 
-$images = [
-    "jpgraph_files/users_logging_on.php?past=day&amp;preceding=hour",
-    "jpgraph_files/users_logging_on.php?past=year&amp;preceding=hour",
-    "jpgraph_files/users_logging_on.php?past=year&amp;preceding=day",
-    "jpgraph_files/users_logging_on.php?past=year&amp;preceding=week",
-    "jpgraph_files/users_logging_on.php?past=year&amp;preceding=fourweek",
-];
-
-foreach ($images as $image) {
-    echo "<img style='max-width: 100%' src='$image'><br>\n";
-}
+echo "<div style='max-width: 640px'>";
+// echo "<div id='past_day_preceding_hour' style='max-height: 400px'></div><hr>";
```

Comment:
Is this intentionally commented out?

Reply:
Instead of echo'ing these out individually, consider using an array+loop (like was there before) so that if we decide to tweak the style or `<hr>` we can do so by editing one line instead of 5.

Reply:
It was since this one chart was a bar chat. I'll uncomment and add the bar chart in.